cfgov/jinja2/v1/index_es.html

```diff
@@ -0,0 +1,183 @@
+{% extends "base.html" %}
+
+{% block title -%}
+    {{ _('Consumer Financial Protection Bureau') }}
+{%- endblock %}
+
+{% block preload %}
+    {{ super() }}
+    <link rel="preload"
+          href="{{ static('img/home-hero_sm.jpg') }}"
+          as="image"
+          media="(max-width: 600px)">
+    <link rel="preload"
+          href="{{ static('img/home-hero_lg.jpg') }}"
+          as="image"
+          media="(min-width: 601px)">
+{%- endblock %}
+
+
+{% block desc -%}
+    Our vision is a consumer finance marketplace that works for American consumers,
```

Comment:
Does this need to be translated?

Reply:
@richaagarwal Yes, that's a good point. I don't think we have that translated yet, so I removed this override. The main desc block defaults to this text but will show anything entered in the description field in the page's admin, so we can start by entering some custom text there. Once we have a translation, we can set it up for the base template and remove the custom text.

Reply:
Just curious: Isn't this only visible if you view source and find the meta description? 
In general, do we need to worry about translating meta values? I'm sure there are people who would appreciate it, but it seems low on the priority scale for translating our site.

Reply:
@higs4281 I think the meta description can appear in search results, so it's not an immediate priority but we might want to document it as a possible future translation.